aws/cloudwatch-sns/variables.tf

```diff
@@ -0,0 +1,39 @@
+variable "enabled" {
+  default = true
+}
+
+variable "region" {
+  default = "us-west-2"
+}
+
+variable "topic_name" {
+  default = "cloudwatch-to-pd"
+}
+
+variable "display_name" {
+  default = ""
+}
+
+variable "kms_master_key_id" {
+  default = "alias/aws/sns"
+}
+
+variable "pagerduty_endpoint_parameter" {
+  description = "SSM parameter store location for pagerduty endpoint"
+  default     = "/cloudwatch_to_pagerduty/endpoint"
+}
+
+variable "pagerduty_endpoint_discover" {
+  description = "Boolean value to decide if we want to discover the value of the pagerduty endpoint, default to true"
+  default     = true
+}
+
+variable "pagerduty_endpoint" {
```

Comment:
If you add sensitive=true here it does a help a little.  Still stored in state file, but at least doesn't spill the secret to your screen every plan/apply.

https://www.terraform.io/docs/configuration/variables.html#suppressing-values-in-cli-output